toqito/matrix_ops/tensor_comb.py

```diff
@@ -53,26 +58,48 @@ def tensor_comb(states: list[np.ndarray], k: int) -> dict:
         :filter: docname in docnames
 
     :raises ValueError: If the input list of states is empty.
-    :param states: A list of quantum state vectors represented as numpy arrays.
-    :param k: The length of the sequence for generating tensor products.
-    :return: A dictionary where:
-
-        - Keys represent sequences (as tuples) of quantum state indices,
-        - Values are density matrices corresponding to the tensor product of
-          the state vectors for the sequence.
-
+    :param k: The length of the sequence.
+    :param states: A list of state vectors.
+    :mode: Determines the type of sequences.
```

Comment:
```suggestion
    :param mode: Determines the type of sequences.
```
You can't have a list in a parameter definition. 

![image](https://github.com/user-attachments/assets/9cce3363-2472-4547-bfce-336a790ee6a4)

Check the build locally to verify everything looks fine in your docstring. Instructions are available in the contributing guide. https://toqito--1153.org.readthedocs.build/en/1153/autoapi/matrix_ops/tensor_comb/index.html